lib/situation_room/check/check_make.ex

```diff
@@ -0,0 +1,25 @@
+defmodule SituationRoom.Check do
+  @moduledoc """
+  Logic for sending requests to site URLs
+  """
+  alias SituationRoom.RequestClient
+
+  def run(url) do
+    start_ms = :os.system_time(:microsecond)
+    res = RequestClient.head(url)
+    time = :os.system_time(:microsecond) - start_ms
+
+    case res do
+      {:ok, response} ->
+        {:ok, %{"status" => response.status, "response_time" => time / 1000}}
```

Comment:
I dont think you need to divide the milliseconds out here.  Keep the timing in milliseconds